core/modules/catalog-next/store/category/actions.ts

```diff
@@ -91,6 +81,33 @@ const actions: ActionTree<CategoryState, RootState> = {
       }
     }
   },
+  /**
+   * Calculates products taxes
+   * Registers URLs
+   * Configures products
+   */
+  async _processCategoryProducts ({ dispatch, rootState }, { products = [], filters = {} } = {}) {
```

Comment:
We never used „_” prefixing in vuex and I guess it would be better to not start any new convention in here - maybe it should be just a helper - if not then Vuex action but with standard name (?)

Reply:
sure, you're right, done :)